bin/SimilarityInterface.py

```diff
@@ -0,0 +1,97 @@
+#notes: many functions are incomplete or disabled because sql isn't implemented yet.
+
+import numpy as np #this does math
+import pandas as pd #this helps with the sorting and the math associated
+import argparse #this helps parse the arguments to the call
+import sys #this is needed to deal with inputs
+
+#sql database to look at
+SQLCommand = "null"
+
+#main function
+def main(args):
+    args = check()
+    simularites = sqlCall(args)  # the table of simularity mesurements
+    sortedSims = sort(simularites)  # the sorted table of simularity mesurements (top = highest)
+    output(sortedSims, args)
+

```

Comment:
I fixed the rest of it and changed the functionality of the threshold inputs to allow more customization of inputs.